alluxio-ui/master/src/store/metrics/reducer.tsx

```diff
@@ -47,6 +50,17 @@ export const metricsReducer: Reducer<IMetricsState> = (state = initialMetricsSta
     case MetricsActionTypes.FETCH_REQUEST:
       return {...state, loading: true};
     case MetricsActionTypes.FETCH_SUCCESS:
+      const timeSeriesMetrics: LineSerieData[] = [];
+      action.payload.data.timeSeriesMetrics.map((item: any) => {
+        // only push the latest 20 points of data
+        timeSeriesMetrics.push({
+          id: item.name,
+          xAxisLabel: 'Time Stamp',
+          yAxisLabel: 'Percent (%)',
```

Comment:
The y axis might not always be percentage

Reply:
Good point, would you like me to keep a map of name to x/y labels? as long as the name does not change, it should work.

Reply:
For now let's leave it as is. I think it would be better to do on the server side.

Reply:
Makes sense